packages/docs/getting-started/running-a-validator.md

```diff
@@ -135,6 +137,15 @@ export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:baklava
 export NETWORK_ID=1101
 ```
 
+For having the Proxy and the Validator reporting to the network EthStats service, you can export the following 2 environment variables:
```

Comment:
I wonder if we should actually just add `--ethstats \<PROXY-NAME\>@baklava-ethstats.celo-testnet.org` directly to the command that is copied and pasted, and mention that you need to substitute the name you used in account registration for `\<PROXY-NAME\>`/`\<VALIDATOR-NAME\>`

I think it could be a bit confusing to have this section at the beginning

Reply:
We should more explicitly indicate where these vars should be defined (i.e. on the validator and proxy machines). I'm inclined to move this to when these would actually get executed